File: spec/code_samples/Python requests/keys@{keyId}@activate/put.py
import requests

url = "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/keys/%7BkeyId%7D/activate"

payload = "{\"allocation\":\"TIER4\"}"
headers = {'content-type': 'application/json'}

response = requests.request("PUT", url, data=payload, headers=headers)

print(response.text)

File: spec/code_samples/Node request/exchange-keys@{keyId}@activate/put.js
var request = require("request");

var options = {
  method: 'PUT',
  url: 'http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/exchange-keys/%7BkeyId%7D/activate',
  headers: {'content-type': 'application/json'},
  body: {allocation: 'TIER4'},
  json: true
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
